examples/high_frequency/emc/double_pulse_test.py

```diff
@@ -0,0 +1,382 @@
+# # Double Pulse Test
+#
+# Description: build the equivalent circuit of a DPT
+#
+# Most examples can be described as a series of steps that comprise a workflow.
+# 1. Import packages and instantiate the application.
+# 2. Insert Circuit Components
+# 3. Create the Wiring
+# 4. Insert simulation set up
+# 3. View the results.
```

Comment:
@tizianrot I would add a simple description of your workflow rather then explain the steps.